### DISCUSSION TITLE
How to get a logo/brand on a single html page in single html file (without folders and multiple files)

### DISCUSSION THREAD

**Discussion by sunvitD** [quarto-dev/quarto-cli#11717] [Q&A]
### Description

I am new to `Quarto`, using it with `python` and its fabulous and loving it so far.  

I am running a `streamlit` webapp and through that I am running `Quarto` that will generate `html` & `pdf` reports dynamically on the user click. Keeping `html` reports in focus as of now,  I am trying to provide a `logo` on `top` of it for the `branding` purposes and the reports to look more `professional` just like any other customer reports having brand & logo on it.

Its not a `website` but just standalone `html` report output that I am generating .

Through the quarto docs and [Stackoverflow post](https://stackoverflow.com/questions/79293484/how-to-display-a-logo-using-brand-yml-within-a-quarto-html-document/) I was able to figure out it would need `_quarto.yml` as well along with `qmd file, _brand.yml, png` and it would be created as `website` as shown in the SO link. I was able to replicate this and it generated `multiple folders and files` in output along with `html file` as it was marked as `website` project.

So is there a way to get **only the html document file output containing logo** in it for the standalone qmd/html files ?

**Comment by mcanouil** [comment]
- https://github.com/quarto-dev/quarto-cli/issues/11627
